packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -136,6 +137,6 @@ export const getStyles = (props: ICheckStyleProps): ICheckStyles => {
       }
     ],
 
-    checkHost: [{}]
+    checkHost: classNames.checkHost
```

Comment:
FYI I'd thought of another way to fix this in #9623 (though this way might be cleaner)

Reply:
Never mind I like this way better